pkg/controller/daemon/update_test.go

```diff
@@ -24,10 +24,10 @@ import (
 )
 
 func TestDaemonSetUpdatesPods(t *testing.T) {
-	manager, podControl, _ := newTestController()
+	ds := newDaemonSet("foo")
```

Comment:
The code now reads from both the cache (lister) and the bare client, so we must include the controller object in both fakes. Previously, many tests only ever populated the cache because that's all that mattered.